packages/adp-tooling/test/unit/writer/__snapshots__/index.test.ts.snap

```diff
@@ -63,18 +62,17 @@ Object {
   "package.json": Object {
     "contents": "{
     \\"name\\": \\"my-test-app\\",
-    \\"version\\": \\"0.1.0\\",
+    \\"version\\": \\"0.1.0\\", 
     \\"private\\": true,
     \\"description\\": \\"Adaptation of the.original.app\\",
     \\"keywords\\": [
         \\"ui5\\",
         \\"sapui5\\",
         \\"adaptation-project\\"
     ],
-    \\"dependencies\\": {},
     \\"devDependencies\\": {
         \\"@sap/ux-ui5-tooling\\": \\"^1.11.4\\",
-        \\"@ui5/cli\\": \\"^3.9.1\\"
+        \\"@ui5/cli\\": \\"^3.6.0\\"
```

Comment:
Just a question, why did this version change? It was `^3.9.1` before. `^3.6.0` will install the latest too.